### DISCUSSION TITLE
Subclustering ArchR project based on Gene Integration Matrix

### DISCUSSION THREAD

**Discussion by LacquerHed** [GreenleafLab/ArchR#1687] [Questions / Documentation] [answered]
I am trying to come up with a good strategy to do this, and I see two potential avenues, just not sure which is more appropriate:

1. Subset my scRNA-seq dataset and ArchrR project by the specific cluster of interest and integrate

2. Subset the total ArchR project for cluster of interest after integration with full scRNA-seq dataset

Thanks for your help.

**Comment by rcorces** [comment, marked as answer]
I dont think there is a single correct answer for this. Option 1 would be fine if you knew that the subcluster in RNA was exactly the same as the subcluster in ATAC and would be suitable for constrained integration. Otherwise I might opt for Option 2
